policyengine_us/parameters/gov/states/id/tax/income/deductions/aged_or_disabled/age_eligibility.yaml

```diff
@@ -0,0 +1,12 @@
+description: Idaho provides the development disabilities deduction for members of the taxpayers at or above this age threshold.
+values:
+  2021-01-01: 65
+metadata:
+  reference:
+    - title: Idaho Title 63, Chapter 30, 63-3022E., (1)
+      href: https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3022e/ 
+    - title: Idaho Form 39R, Resident Supplemental Schedule and Instructions 2022
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EFO00088/EFO00088_12-30-2022.pdf#page=9
```

Comment:
I think there are 2 dependent deductions - one for 1,000 and one for another amount